data.log

```diff
@@ -1,2 +1,5 @@
 http 2015-05-13T23:39:43.945958Z my-loadbalancer 192.168.131.39:2817 10.0.0.1:80 0.000073 0.001048 0.000057 200 200 0 29 "GET http://www.example.com:80/ HTTP/1.1" "curl/7.38.0" - -
-https 2015-05-13T23:39:43.945958Z my-loadbalancer 192.168.131.39:2817 10.0.0.1:80 0.000 0.002 0.000 200 200 145 1396 "GET https://www.example.com:443/ HTTP/1.1" "-" ECDHE-RSA-AES128-GCM-SHA256 TLSv1.2 arn:aws:elasticloadbalancing:us-west-1:123456789012:targetgroup/aaaaa-aaaaa-ABCDEFGHIJKL/1111111111111111 "Root=1-11111111-111111111111111111111111" "www.example.com" "session-reused" 0
\ No newline at end of file
+https 2015-05-13T23:39:43.945958Z my-loadbalancer 192.168.131.39:2817 10.0.0.1:80 0.000 0.002 0.000 200 200 145 1396 "GET https://www.example.com:443/ HTTP/1.1" "-" ECDHE-RSA-AES128-GCM-SHA256 TLSv1.2 arn:aws:elasticloadbalancing:us-west-1:123456789012:targetgroup/aaaaa-aaaaa-ABCDEFGHIJKL/1111111111111111 "Root=1-11111111-111111111111111111111111" "www.example.com" "session-reused" 0 
```

Comment:
double checking - this is the case where we have fewer than `26` fields in the log line?